src/custom/hooks/useAppCode.ts

```diff
@@ -0,0 +1,15 @@
+import { useIsGnosisSafeApp } from 'hooks/useWalletInfo'
+import { DEFAULT_APP_CODE, SAFE_APP_CODE } from 'constants/index'
+
+const APP_CODE = process.env.REACT_APP_APP_CODE
+
+export function useAppCode(): string {
+  const isSafeApp = useIsGnosisSafeApp()
+
+  if (APP_CODE) {
+    // appCode coming from env var has priority
+    return APP_CODE
```

Comment:
This is because we plan not to set it? is just implied by the URLS, but we expect others to set it?

Reply:
Because we can run it without having the APP_CODE env var set, and it'll default to `CowSwap`.